pkg/serving/config_changes_test.go

```diff
@@ -151,6 +151,17 @@ func TestPinImageToDigestNilContainerStatus(t *testing.T) {
 	assert.NilError(t, err)
 }
 
+func TestPinImageToDigestNilContainerSpec(t *testing.T) {
```

Comment:
Does this test belong to this PR?

Reply:
I was trying to improve coverage to prevent the codecov project failure. Removed this test

Reply:
IMO adding unit test to cover this flag in create and update operations would help with coverage. Since the common flag doesn't have its own test file, but are rather tested with those ops.